pkg/ingress/query.go

```diff
@@ -150,3 +158,41 @@ func getRule(host string, path string, pathType *networkingv1.PathType, backend
 	}
 	return rule
 }
+
+func getIngressClass() (string, bool) {
```

Comment:
Could you add a test for this method?

Reply:
This method is to get the default Ingress class.
```suggestion
func getDefaultIngressClass() (string, bool) {
```

Reply:
Hi, actually this method provide default ingressClass if that ingress controller is set to default, if there is no default ingress controller is there then ,it look for nginx ingress controller if available then it add that to jaeger ingress.